src/NUnitFramework/framework/Attributes/DatapointSourceAttribute.cs

```diff
@@ -27,11 +27,10 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// Used to mark a field, property or method providing a set of datapoints to 
-    /// be used in executing any theories within the same fixture that require an 
-    /// argument of the Type provided. The data source may provide an array of
-    /// the required Type or an <see cref="IEnumerable{T}"/>.
-    /// Synonymous with DatapointsAttribute.
+    /// Marks a field, property or method as providing a set of datapoints for use 
+    /// in executing any theories within the same fixture that require an argument 
+    /// of the Type provided. The data source may provide an array of the required 
```

Comment:
For `DatapointsAttribute` we have `... of the provided Type. ...` and not `...of the Type provided. ...` as here. I prefer the `DatapointsAttribute` variant.